### ISSUE TITLE
RFE: wwctl overlay import --recursive

### ISSUE THREAD

**Issue by anderbubble** [hpcng/warewulf#482]
I think it would be very convenient to be able to recursively import a directory into an overlay. e.g., `wwctl overlay import --recursive slurm /etc/slurm`.

**Comment by mslacken** [comment]
No, I think as long as we can have `for file in $(find /etc/slurm -type f); do wwctl overlay import --parent $file; done` A recursive functionality would make more problems than it solves.
